lib/qm-dsp/dsp/tempotracking/DownBeat.cpp

```diff
@@ -207,38 +206,7 @@ DownBeat::findDownBeats(const float *audio,
             oldspec[j] = newspec[j];
         }
     }
-
-    // We now have all spectral difference measures in specdiff
```

Comment:
Where is the deleted code now?

Reply:
That code is in AnalyzerRhythm::ComputeMeter. I added the most outer 'for' there only which hasn't be useful actually, as explained in the comments in the code.
We may consider reverting these changes in qm-dsp if that is a dead end.